### PR TITLE
Fix flatpak-builder-lint warnings

### PR DESCRIPTION
**Pull request by swsnr**
Fixing build errors from https://github.com/flathub/org.viking.Viking/pull/4

### CODE REVIEW

File: org.viking.Viking.yml [hunk 1]
```diff
@@ -157,12 +157,9 @@ modules:
     config-opts:
       - --disable-mapnik
     sources:
-# Note v1.8 doesn't build 'out of the box' due to dependency on gnome-doc-utils
-# Hence creation of v1.8 flatpak version with a few fixes to get build working.
       - type: git
-        url: git://git.code.sf.net/p/viking/code
-#        branch: "viking-1.8-flatpak"
-        branch: "viking-1.10"
+        url: https://git.code.sf.net/p/viking/code
```
Comment (swsnr) [inline org.viking.Viking.yml:155]: Should I use the Github URL here?

**Comment by hfiguiere** [comment]
you should also pickup the changes from #3

**Comment by hfiguiere** [comment]
they would have addressed the failures ahead of time.

**Comment by swsnr** [comment]
So I presume this is not going to be picked up :shrug:

**Pull request closed without merging**